bigquery/cloud-client/README.rst

```diff
@@ -1,149 +1,3 @@
-.. This file is automatically generated. Do not edit this file directly.
+These samples have been moved.
```

Comment:
Can we add where it got moved?

Reply:
It looks like this is accounted for (see the last line of this file)

Reply:
Indeed, the new URL is at the end.

Perhaps it wasn't that apparent in the unified diff view? (a side-by-side diff renders it much more clearly IMHO)